dbus-wait.sh

```diff
@@ -8,7 +8,10 @@ wait_for_dbus() {
 			  /org/freedesktop/DBus \
 			  org.freedesktop.DBus.ListNames
 
-		if [ $? ]; then
+		dbus_test="$?"
```

Comment:
After some reading, I found [this StackOverflow](https://stackoverflow.com/questions/26675681/how-to-check-the-exit-status-using-an-if-statement) to be most helpful. In this case, it would look like:

```
dbus_test=$?
if [ $dbus_test -ne 0 ]; then
```

Reply:
Basically the issue was that when showing "failed to connect to socket" it was still proceeding using the existing code, as shown in the linked comment above:

```
08.10.21 22:01:00 (+0200) helium-miner Failed to open connection to "system" message bus: Failed to connect to socket /host/run/dbus/session_bus_socket: No such file or directory
08.10.21 22:01:00 (+0200) helium-miner DBus is now accepting connections
```

So the idea was to have it so that if it says "can't connect to socket" or returns no output, then it will sleep. Otherwise it will continue.